### DISCUSSION TITLE
Welcome to HuMo-genealogy Discussions!

### DISCUSSION THREAD

**Discussion by HuubMons** [HuubMons/HuMo-genealogy#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HuubMons** [comment]
Test of discussion part at GitHub.

**Comment by Adri1954** [comment]
Hello, very good! I finally found a contact. As already communicated, I am very interested in this project both for practical reasons (a big family to be informed about its origins and developments) and a personal attitude to analysis and programming (at the moment a bit rusty) in php and mysql, and not only. I have a few decades of experience in training and assisting clients on various topics, ranging from experience in the management of information systems for companies to the self-management of a company (now in voluntary liquidation) for the distribution of software with assistance and development. My job was also a hobby, and now I'm a grandfather. What pushes me to participate is having found this project, undoubtedly of a high level (congratulations!), but which I find insufficient to answer the questions that my grandchildren ask: "But who is that gentleman for me?", "Who am I for that lady who is so kind?", etc. For now Humo is not able to answer correctly and peeking at the source of "relationship.php" I understood why. So here I am, available!

**Reply by HuubMons** [reply]
I'm not sure next message was send to you in januari. So just for sure, I send it again:

Hello Adri,

All discussions about HuMo-genealogy can be found at the phpBB forum. This Github discussion platform isn't used a lot.

At this moment I'm trying to rebuild HuMo-genealogy using Bootstrap and MVC model. The latest changes can be found at Github, in the seperate branch.
I will probably release a new version in the next weeks. The menu structure is changed completely.

I see you have experience in software distrubution. At this moment I'm not advertising the program. I actually never did.
The program is made for my own website, and it's just a hobby :-) If there are a lot of users, I can't handle all questions and requests for the program.

**Comment by HuubMons** [comment]
Hello Adri,

All discussions about HuMo-genealogy can be found at the phpBB forum. This Github discussion platform isn't used a lot.

At this moment I'm trying to rebuild HuMo-genealogy using Bootstrap and MVC model. The latest changes can be found at Github, in the seperate branch.
I will probably release a new version in the next weeks. The menu structure is changed completely.

I see you have experience in software distrubution. At this moment I'm not advertising the program. I actually never did.
The program is made for my own website, and it's just a hobby :-) If there are a lot of users, I can't handle all questions and requests for the program.

**Comment by Gawrion** [comment]
Hi there. I'm from Poland and i use Humogen from 2016 for my tree (https://gene.tots.pl). Yesterday i upgraded from version 4.x to newest. I got some feedback and some support (including translation and polish timeline):

1. It was impossible to upgrade directly from 4.x to newest version. Script was trying to upgrade each version but stopped without progress. I was trying to switch PHP to 7.x and 8.x without good results. Then i tried to upgrade slowly through main updates in 4-5 steps and then it worked. It updated database etc.
2. I switched to polish and realized it had a very bad translation. I corrected most website and editor elements, and generated *.po, and *.mo polish files. I also created polish timeline from 900 to 2024 ;). Files attached in zip:
[po-mo-and-polish-timeline.zip](https://github.com/user-attachments/files/17645329/po-mo-and-polish-timeline.zip)

EDIT - dont use polish timeline - it has mistakes as it was generated by GPT and now i check it manually. GPT is still lying too much ;)

U made great tool for genealogy tree. Thank U Man!

BTW. I see Your phpBB is not compliant with GDPR so I can give u my free to use GDPR tool so You can integrate with your phpBB (example of my integration: https://stogi.tots.pl/ucp.php?mode=privacy&sid=5b81d7c5313c08c461ea0dd8b06fbffa). U can register at tool at https://my.oodo.pl - it is ez to configure for persons not familiar to GDPR and create code for integration with website, but if u need support i will help u for free! I'm lawyer-programmer specialized in GDPR from 2015. ;)
We can also talk about some kind of integration of humogen website with my my.oodo.pl so humo gen websites will be GDPR ready.

**Reply by HuubMons** [reply]
Hi Gawrion,

Yes, it's a bit difficult to upgrade from a very old version to a recent version. A lot of changes and updates were done in HuMo-genealogy.
And PHP version was updated from 7 to 8.

I tried using your .po file, but I do have some problems. The first problem is in the zip file. It seems corrupt, I can't open it properly.
Normally all translations are done at the Transifex website: https://app.transifex.com/humo-gen/humo-gen/language/pl_PL/
I tried to upload your updated .po file, but Transifex don't accept the file, there is an error in the file.

I know I have to use a GDPR tool, but in the past PHPBB broke several plugins when I updated PHPBB. At this moment I don't have time to try GDPR scripts. But this will be done in the future.

**Reply by Gawrion** [reply]
Hey - zip was done by 7zip and i changed its original extension, becouse github doesn/t accept 7zip extension - try with 7zip. If it will not work i would do it again ;)
Integration of my.oodo.pl with phpBB is very simple - u only have to generate code in my tool and paste some custom line in phpBB source file (pasting js include). When u have time - give me a signal ;)

**Reply by HuubMons** [reply]
Ah that solves the unzip problem! The .po file is now also added at Transifex. Thank You!

**Reply by Gawrion** [reply]
Good to hear that ;) Thank You!
